stress/client/main.go

```diff
@@ -55,7 +56,8 @@ var (
 	tlsServerName        = flag.String("server_host_override", "foo.test.google.fr", "The server name use to verify the hostname returned by TLS handshake if it is not empty. Otherwise, --server_host is used.")
 	caFile               = flag.String("ca_file", "", "The file containing the CA root cert file")
 
-	logger = grpclog.Component("stress")
+	totalNumCalls int32
```

Comment:
Should this be `int64`?  Or do we think we'll never manage 2B calls?

For reference, 2B microseconds is ~30 minutes.  So if we can do 1 RPC / microsecond (e.g. with parallelism), we'll overflow in half an hour of stress testing.  `int64` seems prudent.

Reply:
Yes, it should be int64, i used a long in Java as well.